### ISSUE TITLE
PowerModels.jl integration: Mismatching LLVM-version

### ISSUE THREAD

**Issue by bergkvist** [e2nIEE/pandapower#1123]
### Trying to import pandapower in python-jl fails (likely because of mismatching LLVM-version)
```
Python 3.8.6 | packaged by conda-forge | (default, Nov 27 2020, 19:44:01) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> import pandapower

signal (11): Segmentation fault
in expression starting at none:4
_ZN4llvm16TargetPassConfig7addPassEPNS_4PassEbb at /opt/conda/lib/python3.8/site-packages/llvmlite/binding/../../../../libLLVM-10.so (unknown line)
unknown function (ip: 0x22)
Allocations: 5044884 (Pool: 5043600; Big: 1284); GC: 6
Segmentation fault
```

### Julia Stable Release uses LLVM 9
```
Julia Version 1.5.3
Commit 788b2c77c1 (2020-11-09 13:37 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, skylake)
```

### Julia Upcoming Release uses LLVM 11
```
Julia Version 1.6.0-beta1
Commit b84990e1ac (2021-01-08 12:42 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.0 (ORCJIT, skylake)
```

### Numba 0.52.0, llvmlite 0.35.0 uses LLVM 10
https://github.com/numba/llvmlite

- Since llvmlite does not yet support LLVM-11, and Julia skipped LLVM-10 - it seems like we have to use LLVM-9.

- Seems like the obvious thing to do here might be to downgrade llvmlite to 0.33.0, and use the Julia stable release.

```
$ conda install llvmlite==0.33.0
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: | 
The environment is inconsistent, please check the package plan carefully
The following packages are causing the inconsistency:

  - conda-forge/noarch::ipympl==0.5.8=pyh9f0ad1d_0
  - conda-forge/noarch::ipywidgets==7.6.2=pyhd3deb0d_0
  - conda-forge/linux-64::widgetsnbextension==3.5.1=py38h578d9bd_4
  - conda-forge/noarch::jupyterlab_server==1.2.0=py_0
  - conda-forge/noarch::jupyterlab==2.2.9=py_0
  - conda-forge/linux-64::ipykernel==5.4.2=py38h81c977d_0
  - conda-forge/linux-64::notebook==6.1.6=py38h578d9bd_0
  - conda-forge/linux-64::ipython==7.19.0=py38h81c977d_0
failed with initial frozen solve. Retrying with flexible solve.
Solving environment: |
```

- And then it keeps working for ~15 minutes before printing out more conflicts than my terminal history can display.

Any ideas?

I'm using jupyter/scipy-notebook:703d8b2dcb88 from Docker Hub.

**Comment by bergkvist** [comment]
Using `conda search --info numba` along with the [compatibility table from llvmlite](https://github.com/numba/llvmlite), I've created a mapping between numba and LLVM-versions:

```
numba 0.36.1 py36hdc42b47_0                     6.0.x
numba 0.36.2 py35_0                             6.0.x
numba 0.36.2 py36_0                             6.0.x
numba 0.38.0 py27_0                             6.0.x
numba 0.38.0 py35_0                             7.0.x
numba 0.38.0 py36_0                             7.0.x
numba 0.38.1 py27_0                             7.0.x
numba 0.38.1 py35_0                             7.0.x
numba 0.38.1 py36_0                             7.0.x
numba 0.40.0 py27h637b7d7_1000                  7.0.x
numba 0.40.0 py27hf8a1672_0                     7.0.x
numba 0.40.0 py35hf8a1672_0                     7.0.x
numba 0.40.0 py36h637b7d7_1000                  7.0.x
numba 0.40.0 py36hf8a1672_0                     7.0.x
numba 0.40.0 py37h637b7d7_1000                  7.0.x
numba 0.40.0 py37hf8a1672_0                     7.0.x
numba 0.40.1 py27h637b7d7_1000                  7.0.x
numba 0.40.1 py27hf8a1672_0                     7.0.x
numba 0.40.1 py36h637b7d7_1000                  7.0.x
numba 0.40.1 py36hf8a1672_0                     7.0.x
numba 0.40.1 py37h637b7d7_1000                  7.0.x
numba 0.41.0 py27h637b7d7_1000                  7.0.x
numba 0.41.0 py27hf8a1672_0                     7.0.x
numba 0.41.0 py36h637b7d7_1000                  7.0.x
numba 0.41.0 py36hf8a1672_0                     7.0.x
numba 0.41.0 py37h637b7d7_1000                  7.0.x
numba 0.41.0 py37hf8a1672_0                     7.0.x
numba 0.42.1 py27hf484d3e_0                     7.0.x
numba 0.42.1 py36hf484d3e_0       7.0.x, 7.1.x, 8.0.x
numba 0.42.1 py37hf484d3e_0       7.0.x, 7.1.x, 8.0.x
numba 0.43.0 py27hf484d3e_0       7.0.x, 7.1.x, 8.0.x
numba 0.43.0 py36hf484d3e_0       7.0.x, 7.1.x, 8.0.x
numba 0.43.0 py37hf484d3e_0       7.0.x, 7.1.x, 8.0.x
numba 0.43.1 py27he1b5a44_0       7.0.x, 7.1.x, 8.0.x
numba 0.43.1 py27hf2d7682_0       7.0.x, 7.1.x, 8.0.x
numba 0.43.1 py36he1b5a44_0       7.0.x, 7.1.x, 8.0.x
numba 0.43.1 py36hf2d7682_0       7.0.x, 7.1.x, 8.0.x
numba 0.43.1 py37he1b5a44_0       7.0.x, 7.1.x, 8.0.x
numba 0.43.1 py37hf2d7682_0       7.0.x, 7.1.x, 8.0.x
numba 0.44.1 py27hb3f55d8_0       7.0.x, 7.1.x, 8.0.x
numba 0.44.1 py36hb3f55d8_0                     9.0.x
numba 0.44.1 py37hb3f55d8_0                     9.0.x
numba 0.45.0 py27hb3f55d8_0                     9.0.x
numba 0.45.0 py36hb3f55d8_0                     9.0.x
numba 0.45.0 py37hb3f55d8_0                     9.0.x
numba 0.45.1 py27hb3f55d8_0                     9.0.x
numba 0.45.1 py36hb3f55d8_0                     9.0.x
numba 0.45.1 py37hb3f55d8_0                     9.0.x
numba 0.46.0 py27hb3f55d8_0                     9.0.x
numba 0.46.0 py27hb3f55d8_1                     9.0.x
numba 0.46.0 py36hb3f55d8_0                     9.0.x
numba 0.46.0 py36hb3f55d8_1                     9.0.x
numba 0.46.0 py37hb3f55d8_0                     9.0.x
numba 0.46.0 py37hb3f55d8_1                     9.0.x
numba 0.47.0 py27hb3f55d8_0                     9.0.x
numba 0.47.0 py36hb3f55d8_0                     9.0.x
numba 0.47.0 py37hb3f55d8_0                     9.0.x
numba 0.47.0 py38hb3f55d8_0                     9.0.x
numba 0.48.0 py36hb3f55d8_0                     9.0.x
numba 0.48.0 py37hb3f55d8_0                     9.0.x
numba 0.48.0 py38hb3f55d8_0                     9.0.x
numba 0.49.0 py36h830a2c2_0                     9.0.x
numba 0.49.0 py37h0da4684_0                     9.0.x
numba 0.49.0 py38hcb8c335_0                     9.0.x
numba 0.49.1 py36h830a2c2_0                     9.0.x
numba 0.49.1 py37h0da4684_0                     9.0.x
numba 0.49.1 py38hcb8c335_0                     9.0.x
numba 0.50.1 py36h830a2c2_0                     9.0.x
numba 0.50.1 py36h830a2c2_1                    10.0.x
numba 0.50.1 py37h0da4684_0                    10.0.x
numba 0.50.1 py37h0da4684_1                    10.0.x
numba 0.50.1 py38hcb8c335_0                    10.0.x
numba 0.50.1 py38hcb8c335_1                    10.0.x
numba 0.51.0 py36h7c3b610_0                    10.0.x
```

**Comment by bergkvist** [comment]
This seems to have fixed my problem:

```
conda install numba==0.49.1
```

Or if using [mamba](https://github.com/mamba-org/mamba), a reimplementation of conda in C++ which is several orders of magnitude faster.
```
mamba install numba==0.49.1
```

My Julia version is 1.5.3, which depends on LLVM 9

numba 0.49.1 seems to depend on llvmlite 0.32.1, which depends on LLVM 8. This seems to defy my assumption that using 2 different versions of LLVM will lead to a crash.

In other words, it works after downgrading numba, but I don't really understand why.
